apollo-biz/src/test/java/com/ctrip/framework/apollo/biz/utils/ConfigChangeContentBuilderTest.java

```diff
@@ -17,49 +17,68 @@
 package com.ctrip.framework.apollo.biz.utils;
 
 import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
-import com.ctrip.framework.apollo.biz.MockBeanFactory;
-import com.ctrip.framework.apollo.biz.entity.Item;
 import org.junit.Before;
 import org.junit.Test;
 
+import com.ctrip.framework.apollo.biz.MockBeanFactory;
+import com.ctrip.framework.apollo.biz.entity.Item;
+
 /**
  * @author jian.tan
  */
 
 public class ConfigChangeContentBuilderTest {
 
-  private final ConfigChangeContentBuilder configChangeContentBuilder = new ConfigChangeContentBuilder();
-  private String configString;
-
-  @Before
-  public void initConfig() {
-
-    Item createdItem = MockBeanFactory.mockItem(1, 1, "timeout", "100", 1);
-    Item updatedItem = MockBeanFactory.mockItem(1, 1, "timeout", "1001", 1);
+	private final ConfigChangeContentBuilder configChangeContentBuilder = new ConfigChangeContentBuilder();
```

Comment:
The code format for this file is not right, please re-format it.

Reply:
sure

Reply:
How can I fix that? I set the format style to the one you provided. Please tell me what's wrong here.

Reply:
After you set the format style, you need to use the ide's built-in `Reformat Code` function to format the changed lines. Then the `tab` characters would be replaced with `space` characters.

```java
	private ConfigChangeContentBuilder configChangeContentBuilder;
```
```java
  private ConfigChangeContentBuilder configChangeContentBuilder;
```

Reply:
Yes, I did that now. Please do check it.